invenio_administration/assets/semantic-ui/js/invenio_administration/src/actions/Actions.js

```diff
@@ -55,26 +62,50 @@ export class Actions extends Component {
             />
           )}
           {displayEdit && (
-            <Button
-              as="a"
-              disabled={disableEdit}
-              href={AdminUIRoutes.editView(listUIEndpoint, resource, idKeyPath)}
-              icon
-              labelPosition="left"
-            >
-              <Icon name="pencil" />
-              Edit
-            </Button>
+            <Popup
+              content={
+                disabledEditMessage
+                  ? disabledEditMessage
+                  : i18next.t("Resource is not editable.")
+              }
+              disabled={!disableEdit}
+              trigger={
+                <span className="mr-5">
+                  <Button
+                    as="a"
+                    disabled={disableEdit}
+                    href={AdminUIRoutes.editView(listUIEndpoint, resource, idKeyPath)}
+                    icon
+                    labelPosition="left"
```

Comment:
```suggestion
                    labelPosition="left"
                    title={disabledEditMessage ? disabledEditMessage : ""}
```

For accessibility :)